pages/api/token/temporary.ts

```diff
@@ -49,18 +50,26 @@ const handler = async (
 ) => {
   try {
     const formID = bearerTokenPayload.formID;
+
     if (formID == undefined) {
       return res.status(403).json({ error: "Invalid form request." });
     }
     if (!process.env.TOKEN_SECRET) {
       return res.status(403).json({ error: "Invalid request." });
     }
+
     const temporaryToken = createTemporaryToken(email, process.env.TOKEN_SECRET);
     await updateTemporaryToken(temporaryToken, email, formID);
+    await sendTemporaryTokenByMail(email, temporaryToken);
```

Comment:
It would be great to have some unit tests (success fail situations) for this method. Also it will help cover the  if condition inside the try-catch.

Reply:
Not sure how this would be useful as the function basically just calls the external library. There is not custom logic in there.